frontend/src/lib-components/molecules/ConfirmedMutation.tsx

```diff
@@ -109,6 +114,7 @@ function ConfirmedMutation_<Arg, Data>(
           text={confirmationText}
           resolveMutation={mutation}
           resolveAction={onClick}
+          resolveDisabled={needsExtraConfirmation && !extraConfirmation}
           resolveLabel={confirmLabel ?? i18n.common.confirm}
           onSuccess={(value) => {
             setConfirming(false)
```

Comment:
Pitäisikö tässä ja `rejectAction`:issa kutsua `setExtraConfirmation(false)`, kun modaali suljetaan?
```suggestion
            setConfirming(false)
            setExtraConfirmation(false)
```

ja alempana
```tsx
rejectAction={() => {
  setConfirming(false)
  setExtraConfirmation(false)
}}
```

Reply:
ah joo, ei näköjään tuukaan mitään unmounttia missä resetoituis automaattisesti